docusaurus/docs/shared/create-plugin.md

```diff
@@ -0,0 +1,36 @@
+Tooling for modern web development can be tricky to wrap your head around. While you certainly can write your own webpack configuration, for this guide, you'll be using grafana create-plugin tool
+
+The Grafana [create-plugin tool](https://www.npmjs.com/package/@grafana/create-plugin) is a CLI application that simplifies Grafana plugin development, so that you can focus on code. The tool scaffolds a starter plugin and all the required configuration for you.
+
+1. In the plugin directory, create a plugin from template using create-plugin. When prompted for the kind of plugin, select `datasource`:
+
+   ```
+   npx @grafana/create-plugin@latest
+   ```
+
+1. Change directory to your newly created plugin:
+
+   ```
+   cd my-plugin
+   ```
+
+1. Install the dependencies:
+
+   ```
+   yarn install
```

Comment:
Similarly not something to fix now, but we should either switch to npm or provide the tab structure for npm/pnpm/yarn as in other sections of the site